client/css/style.css

```diff
@@ -296,6 +297,7 @@ kbd {
 #footer .settings::before { content: "\f013"; /* http://fontawesome.io/icon/cog/ */ }
 #footer .help::before { content: "\f059"; /* http://fontawesome.io/icon/question/ */ }
 
+#form #upload::before { content: "\f093"; /* http://fontawesome.io/icon/upload/ */ }
```

Comment:
Super minor: URL is https://fontawesome.com/icons/upload?style=solid

However, I'm not a huge fan of this icon, not very pretty. I went through the ones FA provide and I think the ones below look better (more or less ordered by preference):

- https://fontawesome.com/icons/paperclip?style=solid
- https://fontawesome.com/icons/cloud-upload-alt?style=solid
- https://fontawesome.com/icons/file-image?style=solid
- https://fontawesome.com/icons/share-square?style=solid
- https://fontawesome.com/icons/desktop?style=solid or https://fontawesome.com/icons/laptop?style=solid (maybe far fetched)

What do you think?

Reply:
`paperclip` makes sense

Reply:
Done